packages/server/migration/1702234882706-rename-question-type-col.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class renameQuestionTypeCol1702234882706 implements MigrationInterface {
+  name = 'renameQuestionTypeCol1702234882706';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
```

Comment:
My concern is why this migration didn't move to new migration files? Do we need any of these fields?

```
 await queryRunner.query(
      `ALTER TABLE "course_model" DROP CONSTRAINT "FK_e04e8cfa5bc6ca85981181a9119"`,
    );
    await queryRunner.query(
      `ALTER TABLE "course_model" DROP COLUMN "organizationCourseId"`,
    );
    await queryRunner.query(
      `ALTER TABLE "organization_course_model" ADD CONSTRAINT "UQ_4fef22be04e7b58e8728a24b207" UNIQUE ("courseId")`,
    );
    await queryRunner.query(
      `ALTER TABLE "organization_course_model" ADD CONSTRAINT "FK_4fef22be04e7b58e8728a24b207" FOREIGN KEY ("courseId") REFERENCES "course_model"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`,
    );
```

Reply:
I am not sure what you mean.